### ISSUE TITLE
Shaky projection

### ISSUE THREAD

**Issue by rooprob** [FolkComputer/folk#185]
Hi,
After getting a nice calibration by keeping the board off the table the resulting outline projections are good.

There is a noticeable shake. Any tips on eliminating this ? 

The projector and camera are locked in a secure stand and there's no breeze to speak of.  

Wondering if there's a processing pipeline update we can make to reduce the camera apriltag edge movement frame to frame to make it less susceptible to digital image noise.

**Comment by osnr** [comment]
Yeah, this is a known problem but we haven't done much about it yet… I think there's probably a lot of tricks that could be tried. Some quick thoughts:

If you want to code up smoothing, you could probably add smoothing either to the whole 3D pose in https://github.com/FolkComputer/folk/blob/main/virtual-programs/tags-to-quads.folk#L564 (you already have the past state accessible there) or to the 2D downprojection in https://github.com/FolkComputer/folk/blob/main/virtual-programs/tags-to-quads.folk#L610

You could also try moving the camera physically closer to the table, adjusting its exposure, or increasing camera capture resolution from 720p default to 1080p (by making/editing ~/folk-live/setup.folk) to try to improve tag capture consistency.

**Comment by rooprob** [comment]
Resolved by https://github.com/FolkComputer/folk/pull/193
